packages/svelte-materialify/@types/Tab.d.ts

```diff
@@ -0,0 +1,12 @@
+import { SvelteComponent } from './shared';
+
+export interface TabProps {
+    class?: string;
+    value?: any;
+    activeClass?: any;
+    disabled?: object;
```

Comment:
boolean (if you want, you can change Tab.svelte to be disabled=false (instead of disabled=null))